doc/api/readline.md

```diff
@@ -369,7 +369,7 @@ changes:
   * `crlfDelay` {number} If the delay between `\r` and `\n` exceeds
     `crlfDelay` milliseconds, both `\r` and `\n` will be treated as separate
     end-of-line input. Default to `100` milliseconds.
-    `crlfDelay` will be coerced to `[100, 2000]` range.
+    `crlfDelay` will be coerced to a number no less than `100`. It can be set to `Infinity`, in which case `\r` followed by `\n` will always be considered a single newline.
```

Comment:
Sorry to comment again, but it would be great if you could wrap this line around at 80 characters … if not, that can also be done while landing the PR